airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.tsx

```diff
@@ -1,191 +1,180 @@
-import { faQuestionCircle } from "@fortawesome/free-regular-svg-icons";
+import { faSlack } from "@fortawesome/free-brands-svg-icons";
+import { faEnvelope } from "@fortawesome/free-regular-svg-icons";
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import React from "react";
 import { FormattedMessage, FormattedNumber } from "react-intl";
 import { NavLink } from "react-router-dom";
-import styled from "styled-components";
+import { useIntercom } from "react-use-intercom";
```

Comment:
Can we instead use the previous provider from the cloud package instead of the `react-use-intercom` one directly, since this one has a different behavior.